roles/openshift_storage_glusterfs/tasks/heketi_deploy_part2.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Create heketi DB volume
+  command: "heketi-cli -s http://{{ openshift_storage_glusterfs_heketi_url }} setup-openshift-heketi-storage --listfile {{ mktemp.stdout }}/heketi-storage.json"
+  register: setup_storage
+  failed_when: False
+
+- name: Copy heketi DB to GlusterFS volume
+  command: "oc create -f {{ mktemp.stdout }}/heketi-storage.json"
```

Comment:
I believe it can't be because `heketi-storage.json` is a List of objects, and you can't use `oc_obj` to create without a name. I'll add the namespace, though.